### DISCUSSION TITLE
self deployment with heroku

### DISCUSSION THREAD

**Discussion by Grumpypjs08** [Budibase/budibase#1011] [Help] [answered]
How do i/can I deploy on Heroku? Is there a deployment link?

**Comment by joebudi** [comment]
Hey!

We're working on this at the moment. It shouldn't be long before we have it done (week or so).

**Comment by shogunpurple** [comment, marked as answer]
@Grumpypjs08 we have decided not to pursue heroku for the time being. You can however deploy very easily to Digitalocean, by following this guide: https://docs.budibase.com/self-hosting/hosting-methods/digitalocean

**Reply by Grumpypjs08** [reply]
Is there any way to do this for free?

**Reply by shogunpurple** [reply]
Yep, if you read the tutorial you can use our referral code to get $100 of free credit for Digitalocean, so you can host for free.

**Comment by Grumpypjs08** [comment]
Is there a reason? Or any other way?

**Comment by ramonck** [comment]
I am also +1 for heroku.

**Reply by mjashanks** [reply]
This won't be a priority for us - AWS, Azure and GCP would have to come first, and we're not even sure when that will happen!

Would be delighted to see this as a contribution from anyone who has Heroku experience!

**Reply by ramonck** [reply]
The problem is that it would be required changed to your package.json I can try having a look when I have time.

Here is how it works.
https://devcenter.heroku.com/articles/heroku-button
